scripts/touched.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import os
+import json
+
+query = """
+{
+  repository(name: "website", owner: "kubernetes") {
+    pullRequests(labels: "language/fr", last: 100) {
```

Comment:
`language/fr` can be a script parameter, so other localization teams can use it.

You can use [Click](https://click.palletsprojects.com/en/7.x/) to have a more user-friendly CLI.

Reply:
Done